pkg/preparer/orchestrate.go

```diff
@@ -418,7 +420,9 @@ func (p *Preparer) installAndLaunchPod(pair ManifestPair, pod Pod, logger loggin
 			}
 		} else {
 			// TODO: do this in a transaction
-			err = p.podStore.WriteRealityIndex(pair.PodUniqueKey, p.node)
+			// TODO dai - what do I do with the errors?
```

Comment:
it's worth noting that most of these functions will only return an error if `transaction.Add` fails which should only happen if we've passed the 64 operation limit that consul imposes on transactions. We're not even close to hitting that limit here.

The `Commit()` part could fail if there is an intermittent issue talking to consul. we could consider just putting retries around that part